lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.30",
```

Comment:
You don't need to update version manually. Lerna does this on `lerna publish`

Reply:
hmmm wired I haven't touched this file manually.